### DISCUSSION TITLE
Object identification problem. How do I determine that I have already seen this object? What models can you recommend?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [ultralytics/yolov5#9921] [Q&A]
What is the best mode to identify an object? (some generic object - like a car or something like that).
The goal is to say  - if a model have already seen this object before.
What are the best models and solutions for this problem? 
Preferable something fast))

Are there any generic solutions or do I have to train each separate model for each separate class? 
Can someone share his/her own experience?

Let's say has it ever been successfully achieved or is it a scientific study?

**Comment by glenn-jocher** [comment]
You would train one YOLOv5 model on all the classes you need.
